styles.css

```
@@ -1,797 +1,1357 @@
+  --shadow-lg: 0 8px 24px rgba(0,0,0,0.08);
+  
+  /* Status */
+  --success: #22c55e;
+  --warning: #f59e0b;
+  --danger: #ef4444;
+  
+  /* Layout */
+  --header-height: 56px;
+  --footer-height: 50px;
+  --radius: 12px;
+  --radius: 12px;
```

Biome (2.1.2)

[error] Duplicate custom properties can lead to unexpected behavior and may override previous declarations unintentionally.

--radius is already defined here.

Remove or rename the duplicate custom property to ensure consistent styling.

(lint/suspicious/noDuplicateCustomProperties)